playground/backend/internal/db/datastore/datastore_db_test.go

```diff
@@ -439,6 +443,9 @@ func TestDatastore_GetCatalog(t *testing.T) {
 				t.Errorf("GetCatalog() error = %v, wantErr %v", err, tt.wantErr)
 			}
 			if err == nil {
+				if len(catalog) != 1 {
```

Comment:
@TSultanov  I think I would create a new separate test or table test entry for that and also please ensure that you delete from the datastore emulators (see cleanData func in the test)

Reply:
I don't think that making a separate test makes much sense here - it will have exactly the same prepare function and the assertion case is quite simple